Userland/Applications/Maps/main.cpp

```diff
@@ -20,14 +21,16 @@
 #include <LibGUI/Window.h>
 
 static int constexpr MAP_ZOOM_DEFAULT = 3;
+static int constexpr FAVORITE_MAP_ZOOM = 17;
```

Comment:
Maybe remember the actual zoom level in which the favorite was made?

Reply:
Yes that is better behaviour